zulipterminal/config/keys.py

```diff
@@ -200,6 +200,11 @@
         'help_text': 'View stream description',
         'key_category': 'stream_list',
     }),
+    ('USER_INFO', {
```

Comment:
It would also benefit to have a note for this in `README`. (Maybe these two changes can be extracted into a separate commit)